extensions/extensions-helm/aissemble-spark-history-chart/values.yaml

```diff
@@ -0,0 +1,58 @@
+# This file contains the default values for the aiSSEMBLE Spark Operator Helm chart.
+
+enable: true
+
+deployment:
+  annotations: {}
+  labels: {}
+  replicas: 1
+  image:
+    repository: "apache/spark"
+    imagePullPolicy: IfNotPresent
+    tag: "3.5.1"
+  command: ["/opt/spark/sbin/start-history-server.sh"]
+  env:
+    SPARK_NO_DAEMONIZE: "true"
+  envFromSecret: {}
+  volumes: []
+  volumeMounts: []
+
+dependencies:
+  packages: []
+  jars: []
+
+service:
+  annotations: {}
+  type: LoadBalancer
+  port:
+    name: "shs-http"
+    port: 18080
+
+ingress:
+  enabled: true
+  metadata:
+    annotations: {}
+  ingressClassName: nginx
+  hosts:
+    - paths:
+        - path: /
+          pathType: Prefix
+          backend:
+            service:
+              name: spark-history-service
```

Comment:
Since the name of the service will always be `{{ .Release.Name }}-service`, there's no point in being able to set it here. We know what the name will be in the ingress template right?

Reply:
Actually hear me out-- We accept a list of ingress hosts and paths (do we need to?), and hardcoding it to .Release.Name in the template would force _all_ ingress hosts and paths to point at the same service.  The question is, do we actually need to support multiple ingresses out of the box, or should we just support the expected one?

Reply:
I think that makes sense to accept a list of ingress and point them all to the same service

Reply:
I feel less confident that it does make sense, but I'm not sure.  Forcing it all to the same service seems like it would negate the benefit of allowing multiple ingress paths.

Reply:
I almost want to hardcode _just_ the default ingress, and say that any supplemental ingresses have to be made by hand.

Reply:
When would you want an ingress to not point at your service? In any case, maybe we should just make the ingress service default to `{{ .Release.Name }}-service` with the option to be overrriden.

Reply:
I think that makes a lot of sense.

Reply:
Done-- Take a look!

Reply:
Looks good!